### PR TITLE
Proof point threat response

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/28954

## Description
fixed an issue where command  'proofpoint-tr-update-incident-comment' was missing, name was mismatched.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py [hunk 1]
```diff
@@ -721,7 +721,7 @@ def main():
     elif command == 'proofpoint-tr-get-incident':
         get_incident_command()
 
-    elif command == 'proofpoint-tr-add-comment-to-incident':
+    elif command == 'proofpoint-tr-update-incident-comment':
```
Comment (ronykoz) [inline Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py:724]: are you sure it's the right re-name?

Comment (abaumgarten) [reply Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py:724]: That's what was written in the issue. I saw in addition that there is no implementation for the command proofpoint-tr-update-incident-comment that appears in the documentation commands list

Comment (guykeller) [reply Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py:724]: Let's ask whoever wrote this integration what should be the correct name, or look at the API.
Does it add an incident comment, or updates one?

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/ProofpointThreatResponse/ReleaseNotes/1_0_2.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Proofpoint Threat Response (Beta)
+Maintenance and stability enhancements.
```
Comment (guykeller) [inline Packs/ProofpointThreatResponse/ReleaseNotes/1_0_2.md:4]: I think on this occasion we need more detailed release notes

File: Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py (hunk 1 above)
Comment (abaumgarten) [reply Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py:724]: @yuvalbenshalom @liorblob Which one of you wrote that integration? 
Do you think that proofpoint-tr-update-incident-comment is the correct command? it was missing before .

File: Packs/ProofpointThreatResponse/ReleaseNotes/1_0_2.md (hunk 2 above)
Comment (abaumgarten) [reply Packs/ProofpointThreatResponse/ReleaseNotes/1_0_2.md:4]: Done

**Review by guykeller — APPROVED** [review]

File: Packs/ProofpointThreatResponse/ReleaseNotes/1_0_2.md (hunk 2 above)
Comment (guykeller) [reply Packs/ProofpointThreatResponse/ReleaseNotes/1_0_2.md:4]: Not seeing the  changes

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/ProofpointThreatResponse/ReleaseNotes/1_0_2.md (hunk 2 above)
Comment (abaumgarten) [reply Packs/ProofpointThreatResponse/ReleaseNotes/1_0_2.md:4]: Somehow it was not appear here. Did push again. (you can see it below)

**Review by guykeller — APPROVED** [review]

**Pull request merged**